<!DOCTYPE html>
<!-- saved from url=(0053)file:///Users/gareth/Google%20Drive/lens-n/index.html -->
<html xmlns:mml="http://www.w3.org/1998/Math/MathML"><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
    <title>NofE Reader - PubMed</title>
    <link href="./repeatseq_files/css" rel="stylesheet" type="text/css">

    <link rel="stylesheet" type="text/css" media="all" href="./repeatseq_files/font-awesome.min.css">

    <!-- A combined lens.css will be generated in the bundling process -->
    <!-- While in development, separate links for each CSS file are added, so we don't need a source map -->
    <script src="./repeatseq_files/jquery.min.js"></script>
    <link  rel="stylesheet" href="../css/reader.css">
    <!-- A combined lens.css will be generated in the bundling process -->
    <!-- While in development, separate links for each CSS file are added, so we don't need a source map -->
    <link href="../lib/lens.css" rel="stylesheet" type="text/css">


    <!-- styles needed by jScrollPane -->
    <link type="text/css" href="../lib/jquery.jscrollpane.css" rel="stylesheet" media="all" />

    <!-- the mousewheel plugin - optional to provide mousewheel support -->
    <script type="text/javascript" src="../lib/jquery.mousewheel.js"></script>

    <!-- the jScrollPane script -->
    <script type="text/javascript" src="../lib/jquery.jscrollpane.js"></script>

    <script src="../reader.js"></script>
    <!-- MathJax Configuration -->
    <script type="text/x-mathjax-config;executed=true">
      MathJax.Hub.Config({
        jax: ["input/TeX", "input/MathML","output/HTML-CSS"],
        SVG: { linebreaks: { automatic: true }, EqnChunk: 9999  },
        "displayAlign": "left",
        styles: {".MathJax_Display": {padding: "0em 0em 0em 3em" },".MathJax_SVG_Display": {padding: "0em 0em 0em 3em" }}
      });
    </script>
    <script type="text/javascript" src="./repeatseq_files/MathJax.js"></script>

    <script src="./repeatseq_files/lens.js"></script>

    <script>

      // Little helper used to parse query strings from urls
      // --------
      //
      // This is needed to support switching to other documents using
      // a url fragment `?url=http://path.to/jats-conforming.xml`

      var qs = function () {
        var query_string = {};
        var query = window.location.search.substring(1);
        var vars = query.split("&");
        for (var i=0;i<vars.length;i++) {
          var pair = vars[i].split("=");
            // If first entry with this name
          if (typeof query_string[pair[0]] === "undefined") {
            query_string[pair[0]] = pair[1];
            // If second entry with this name
          } else if (typeof query_string[pair[0]] === "string") {
            var arr = [ query_string[pair[0]], pair[1] ];
            query_string[pair[0]] = arr;
            // If third or later entry with this name
          } else {
            query_string[pair[0]].push(pair[1]);
          }
        }
        return query_string;
      }();
      /*
      var documentURL = "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&name=pubchase&email=alexei@zappylab.com&retmode=xml&id=PMC3592458";
//      var documentURL = "https://s3.amazonaws.com/elife-cdn/elife-articles/00778/elife00778.xml";

      $(function() {

        // Create a new Lens app instance
        // --------
        //

        var app = new window.Lens({
          document_url: qs.url ? decodeURIComponent(qs.url) : documentURL
        });

        // And start it
        // --------
        //
        // The app will be injected into the body element

        app.start();

        window.app = app;
      });
      */
    </script>
  <style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Hover_Arrow {position: absolute; width: 15px; height: 11px; cursor: pointer}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; color: #666666}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_Menu_Close {position: absolute; width: 31px; height: 31px; top: -15px; left: -15px}
</style><style type="text/css">.MathJax_Preview .MJXc-math {color: inherit!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
.MathJax_Display {padding: 0em 0em 0em 3em}
.MathJax_SVG_Display {padding: 0em 0em 0em 3em}
</style><style type="text/css">.MJXc-script {font-size: .8em}
.MJXc-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXc-bold {font-weight: bold}
.MJXc-italic {font-style: italic}
.MJXc-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXc-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXc-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXc-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXc-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXc-largeop {font-size: 150%}
.MJXc-largeop.MJXc-int {vertical-align: -.2em}
.MJXc-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXc-display {display: block; text-align: center; margin: 1em 0}
.MJXc-math span {display: inline-block}
.MJXc-box {display: block!important; text-align: center}
.MJXc-box:after {content: " "}
.MJXc-rule {display: block!important; margin-top: .1em}
.MJXc-char {display: block!important}
.MJXc-mo {margin: 0 .15em}
.MJXc-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXc-denom {display: inline-table!important; width: 100%}
.MJXc-denom > * {display: table-row!important}
.MJXc-surd {vertical-align: top}
.MJXc-surd > * {display: block!important}
.MJXc-script-box > *  {display: table!important; height: 50%}
.MJXc-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXc-script-box > *:last-child > * {vertical-align: bottom}
.MJXc-script-box > * > * > * {display: block!important}
.MJXc-mphantom {visibility: hidden}
.MJXc-munderover {display: inline-table!important}
.MJXc-over {display: inline-block!important; text-align: center}
.MJXc-over > * {display: block!important}
.MJXc-munderover > * {display: table-row!important}
.MJXc-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXc-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXc-mtr {display: table-row!important}
.MJXc-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXc-mtr > .MJXc-mtd:first-child {padding-left: 0}
.MJXc-mtr:first-child > .MJXc-mtd {padding-top: 0}
.MJXc-mlabeledtr {display: table-row!important}
.MJXc-mlabeledtr > .MJXc-mtd:first-child {padding-left: 0}
.MJXc-mlabeledtr:first-child > .MJXc-mtd {padding-top: 0}
.MJXc-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXc-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXc-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXc-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXc-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXc-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXc-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXc-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXc-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXc-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXc-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_CHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>
  <body class="current-view reader"><div id="MathJax_Message" style="display: none;"></div><div id="container"><div style="display: none;" class="browser-not-supported">Sorry, your browser is not supported.</div><div class="spinner-wrapper"><div class="spinner"></div><div class="message">Typesetting</div></div><div id="main" class="" style="opacity: 1;"><div class="article lens-article" data-context="toc"><div class="panel content document"><div class="surface resource-view content"><div class="nodes"><div class="content-node cover" data-id="cover"><div class="content"><div class="published-on">January 2013 in <i>Nucleic Acids Research</i></div><span class="text title">Accurate human microsatellite genotypes from high-throughput resequencing data using informed error profiles</span><div class="authors"><div class="content-node text" data-id="text_contributor_1_reference"><div class="content"><a href="" data-id="contributor_reference_1" class="annotation contributor_reference resource-reference">Gareth Highnam</a></div><div class="focus-handle"></div></div><div class="content-node text" data-id="text_contributor_2_reference"><div class="content"><a href="" data-id="contributor_reference_2" class="annotation contributor_reference resource-reference">Christopher Franck</a></div><div class="focus-handle"></div></div><div class="content-node text" data-id="text_contributor_3_reference"><div class="content"><a href="" data-id="contributor_reference_3" class="annotation contributor_reference resource-reference">Andy Martin</a></div><div class="focus-handle"></div></div><div class="content-node text" data-id="text_contributor_4_reference"><div class="content"><a href="" data-id="contributor_reference_4" class="annotation contributor_reference resource-reference">Calvin Stephens</a></div><div class="focus-handle"></div></div><div class="content-node text" data-id="text_contributor_5_reference"><div class="content"><a href="" data-id="contributor_reference_5" class="annotation contributor_reference resource-reference">Ashwin Puthige</a></div><div class="focus-handle"></div></div><div class="content-node text" data-id="text_contributor_6_reference"><div class="content"><a href="" data-id="contributor_reference_6" class="annotation contributor_reference resource-reference">David Mittelman</a></div><div class="focus-handle"></div></div><div class="content-node text plain"><div class="content"></div></div></div><div class="doi">DOI: <a href="http://dx.doi.org/10.1093/nar/gks981">10.1093/nar/gks981</a></div></div><div class="focus-handle"></div></div><div class="content-node heading level-1" data-id="heading_1"><div class="content"><span class="text title">Abstract</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_11"><div class="content"><div class="content-node text" data-id="text_11"><div class="content">Repetitive sequences are biologically and clinically important because they can influence traits and disease, but repeats are challenging to analyse using short-read sequencing technology. We present a tool for genotyping microsatellite repeats called RepeatSeq, which uses Bayesian model selection guided by an empirically derived error model that incorporates sequence and read properties. Next, we apply RepeatSeq to high-coverage genomes from the 1000 Genomes Project to evaluate performance and accuracy. The software uses common formats, such as VCF, for compatibility with existing genome analysis pipelines. Source code and binaries are available at <a href="http://github.com/adaptivegenome/repeatseq" data-id="link_1_divider_undefined" class="link">http://github.com/adaptivegenome/repeatseq</a>.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-1" data-id="heading_4"><div class="content"><span class="text title">Main Text</span></div><div class="focus-handle"></div></div><div class="content-node heading level-2" data-id="heading_5"><div class="content"><span class="text title">INTRODUCTION</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_15"><div class="content"><div class="content-node text" data-id="text_15"><div class="content">Tandem repeats are dispersed throughout the genome, in and around gene regions. They were first identified as agents of disease &#8764;20 years ago (<a href="" data-id="citation_reference_1_divider_article_citation_1" class="annotation citation_reference resource-reference">1</a>), and since then, several microsatellite repeats (not all of which are triplets) have been identified as the underlying basis for a wide range of neurological and morphological disorders in humans and other mammals (<a href="" data-id="citation_reference_2_divider_article_citation_2" class="annotation citation_reference resource-reference">2�4</a>). In addition to causing disease, microsatellites can exert subtle effects on gene function and quantitative traits [reviewed in (<a href="" data-id="citation_reference_3_divider_article_citation_5" class="annotation citation_reference resource-reference">5</a>)]. Repeats are also mutational hotspots in that their instability can be triggered by nearly any aspect of DNA metabolism, and even transcription or stress [reviewed in (<a href="" data-id="citation_reference_4_divider_article_citation_6" class="annotation citation_reference resource-reference">6</a>)]. This sensitivity to defects in repair and cellular insults makes repeats useful markers for genome instability and cancer (<a href="" data-id="citation_reference_5_divider_article_citation_7" class="annotation citation_reference resource-reference">7</a>,<a href="" data-id="citation_reference_6_divider_article_citation_8" class="annotation citation_reference resource-reference">8</a>). Further, analysing repeats in personal genomes promises benefit not just to medical genetics and the diagnosis of repeat-related disorders but also to forensics and genealogy, where shorter and more stable tandem repeats can serve as DNA fingerprints to uniquely identify individuals (<a href="" data-id="citation_reference_7_divider_article_citation_9" class="annotation citation_reference resource-reference">9</a>,<a href="" data-id="citation_reference_8_divider_article_citation_10" class="annotation citation_reference resource-reference">10</a>). The use of accurately and globally measuring tandem repeats spans medicine, genetics and biotechnology; repeats influence clinical and subclinical phenotypes, are signatures for genomic instability and cancer and are important markers for forensics and genealogy.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_16"><div class="content"><div class="content-node text" data-id="text_16"><div class="content">Despite their use and biological importance, some repetitive sequences (particularly microsatellites) are challenging to study with short-read sequencing technology. Genotyping microsatellite repeats from reference-mapped reads is fundamentally distinct from calling SNPs or indels in non-repetitive sequence because there is no sound basis for inferring homology between pairs of aligned repeat units. Microsatellite genotypes must be assigned in terms of allele length or the number of sequenced bases within a read separating the non-repetitive flanking boundaries aligned to the reference, irrespective of intervening alignment gaps. Furthermore, reads must span an entire repeat track to confidently support an allele length. We recently demonstrated the advantage of this repeat-aware genotyping approach in a population-scale analysis of &gt;100 inbred isolates from the <span data-id="emphasis_5_divider_undefined" class="annotation emphasis">Drosophila</span> Genetics Reference Panel (DGRP) (<a href="" data-id="citation_reference_9_divider_article_citation_11" class="annotation citation_reference resource-reference">11</a>,<a href="" data-id="citation_reference_10_divider_article_citation_12" class="annotation citation_reference resource-reference">12</a>). Here, we describe a comprehensive genotyping software package that we have termed RepeatSeq, for calling microsatellite repeat genotypes from whole genome resequencing data. RepeatSeq combines a repeat-aware method for repeat allele determination (<a href="" data-id="citation_reference_11_divider_article_citation_11" class="annotation citation_reference resource-reference">11</a>) with a Bayesian genotyping approach that uses a novel error model informed by properties of a repeat sequence and the reads that map to it.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_17"><div class="content"><div class="content-node text" data-id="text_17"><div class="content">Current approaches for identifying repeat mutations include indel genotyping methods implemented in popular software suites, such as GATK (<a href="" data-id="citation_reference_12_divider_article_citation_13" class="annotation citation_reference resource-reference">13</a>) and ATLAS2 (<a href="" data-id="citation_reference_13_divider_article_citation_14" class="annotation citation_reference resource-reference">14</a>), that can reveal indels within repeat regions or the recently reported lobSTR method (<a href="" data-id="citation_reference_14_divider_article_citation_10" class="annotation citation_reference resource-reference">10</a>), which much like our approach, considers repeats in their entirety. Indel callers are ill-suited for identifying repeat mutation, as they do not report repeat genotypes, they can base indel identification on reads that do not fully span the repeat and they fail to account for the error rates of different repeat types. The mutation rate of microsatellite repeats is influenced largely not only by the length of repeat tract but also by other intrinsic properties, such as the size of the repeated unit and the purity (lack of interruptions) of the repeated sequence (<a href="" data-id="citation_reference_15_divider_article_citation_15" class="annotation citation_reference resource-reference">15</a>). A genotyping method that incorporates the mutational properties of repeat sequences will be better able to distinguish false alleles from true heterozygosity. However, the success of a genotyping approach relies on more than just the accurate identification of true alleles�the method must be applicable to the greatest number of loci genome-wide. The lobSTR method, for example, makes microsatellite calls genome-wide (<a href="" data-id="citation_reference_16_divider_article_citation_10" class="annotation citation_reference resource-reference">10</a>); however, it is blind to homopolymers runs (i.e. mononucleotide repeats, which are a common and important source of genetic variation). In the following sections we describe the RepeatSeq algorithm and error model and its application to high-coverage genomes from the 1000 Genomes Project (<a href="" data-id="citation_reference_17_divider_gks981-B16" class="annotation citation_reference resource-reference">16</a>). RepeatSeq assigned genotypes to &#8764;90% of the repeat loci, including homopolymers. In addition, comparing RepeatSeq to lobSTR revealed that &#8764;90% of repeats genotyped by both methods were assigned the same call. The software inputs high-throughput resequencing data in BAM format (<a href="" data-id="citation_reference_18_divider_article_citation_17" class="annotation citation_reference resource-reference">17</a>) and outputs genotypes in multiple formats, including VCF format (<a href="" data-id="citation_reference_19_divider_article_citation_18" class="annotation citation_reference resource-reference">18</a>), for compatibility and easy integration with existing genotyping pipelines.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-2" data-id="heading_8"><div class="content"><span class="text title">METHODS</span></div><div class="focus-handle"></div></div><div class="content-node heading level-3" data-id="heading_6"><div class="content"><span class="text title">Program outline</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_18"><div class="content"><div class="content-node text" data-id="text_18"><div class="content">The genotyping process is summarized in <a href="" data-id="figure_reference_1_divider_figure_1" class="annotation figure_reference resource-reference">Figure 1</a>. Reads are first mapped to a reference sequence and then sorted, with duplicate reads marked. Next, reads mapping to reference repeat regions are locally realigned using the GATK IndelRealigner tool. Then, for each repeat, the RepeatSeq module discards reads that do not completely span the repeat, as these reads cannot unambiguously support a repeat allele length. The extent to which a read must overlap a repeat (and match the reference) is an adjustable parameter, although we have previously optimized this value to maximize the number of repeat regions that could be genotyped while minimizing the inclusion of improperly mapped reads (<a href="" data-id="citation_reference_20_divider_article_citation_11" class="annotation citation_reference resource-reference">11</a>). Finally, RepeatSeq assigns the most probable genotype using a fully Bayesian approach and considers the reference length of the repeat, the repeat unit size and the average base quality of the mapped reads, as previous information. RepeatSeq implements a diploid genotyping model by default, but it can be set at runtime to implement a haploid model.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_19"><div class="content"><div class="content-node text" data-id="text_19"><div class="content">Several other read filters can be adjusted. For example, reads can be excluded from consideration on the basis of their length, mapping quality and on whether they are properly paired. By default, RepeatSeq returns microsatellite calls in VCF format for compatibility and interoperability with other indel callers. However, RepeatSeq can also output a detailed report that returns annotated alignments of reads in addition to just the assigned genotypes.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-3" data-id="heading_7"><div class="content"><span class="text title">Probabilistic determination of repeat genotypes</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_20"><div class="content"><div class="content-node text" data-id="text_20"><div class="content">The probability that a read is representative of a true allele can be inferred by the properties of the read and the reference repeat sequence that it maps to (<a href="" data-id="citation_reference_21_divider_article_citation_11" class="annotation citation_reference resource-reference">11</a>). We explored many such properties in a previous DGRP population study (<a href="" data-id="citation_reference_22_divider_article_citation_12" class="annotation citation_reference resource-reference">12</a>). The DGRP is composed of &gt;100 fly genomes; each derived from single female founders of a natural fly population and bred to near-isogeny by 20 generations of full-sibling matings. Therefore, in the absence of mapping, alignment or sequence errors, all reads from a single inbred line mapped to a specific microsatellite locus should possess the same repeat allele length, and this homozygosity permits the assumption that deviant reads represent error. To strengthen the validity of this assumption, we minimized the effects of any residual heterozygosity by excluding genomic regions with &gt;5% heterozygosity (based on the SNP calls). For loci to which at least 10 reads had been mapped, reads that supported the majority allele were assumed to support the true allele at the locus, whereas reads that supported a non-majority allele were assumed to be errors. The proportions of reads supporting majority alleles were computed within five bins of equal size for the following properties: reference repeat length, repeat unit size and average base quality for the read. The resulting 5 � 5 � 5 array reflects the frequency of discordant and concordant reads and is the source of error profiles for RepeatSeq.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_21"><div class="content"><div class="content-node text" data-id="text_21"><div class="content">Once reads are mapped, realigned and then pruned of those that do not span the repeat, RepeatSeq calls the most probable genotype using Bayesian model selection. The error rate (<span data-id="inline-formula_5_divider_formula_5" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i3.jpg"></div></span>) is populated with prior information by associating the reference repeat length, unit size and average base quality of the read data with the appropriate bin of the error profiles. The reads at a given locus support <span data-id="inline-formula_6_divider_formula_6" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i4.jpg"></div></span> length variants <span data-id="inline-formula_7_divider_formula_7" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i5.jpg"></div></span> where <span data-id="inline-formula_8_divider_formula_8" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i6.jpg"></div></span>. RepeatSeq considers each homozygous and heterozygous genotype suggested by the read data, of which, there are <span data-id="inline-formula_9_divider_formula_9" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i7.jpg"></div></span>. For example, if two length alleles are present then the heterozygous genotype of the two variants and each homozygous genotype are considered. If three variants are present, then the possible three homozygous and three heterozygous genotypes are considered.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_22"><div class="content"><div class="content-node text" data-id="text_22"><div class="content">Let <span data-id="inline-formula_10_divider_formula_10" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i8.jpg"></div></span> denote the reads at a given locus. Denote the <span data-id="inline-formula_11_divider_formula_11" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i9.jpg"></div></span> possible genotypes <span data-id="emphasis_6_divider_undefined" class="annotation emphasis">g</span><span data-id="subscript_1_divider_undefined" class="annotation subscript">1</span>, <span data-id="emphasis_7_divider_undefined" class="annotation emphasis">g</span><span data-id="subscript_2_divider_undefined" class="annotation subscript">2</span>, <span data-id="inline-formula_12_divider_formula_12" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i10.jpg"></div></span>. For heterozygous and homozygous genotypes, we assume the likelihood <span data-id="inline-formula_13_divider_formula_13" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i11.jpg"></div></span> is multinomial <span data-id="inline-formula_14_divider_formula_14" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i12.jpg"></div></span>, and <span data-id="inline-formula_15_divider_formula_15" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i13.jpg"></div></span> is Dirichlet<span data-id="inline-formula_16_divider_formula_16" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i14.jpg"></div></span> , where the elements of <span data-id="inline-formula_17_divider_formula_17" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i15.jpg"></div></span> sum to unity. For the purpose of elucidating the statistical model, let <span data-id="inline-formula_18_divider_formula_18" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i16.jpg"></div></span> generically denote a homozygous genotype and let <span data-id="inline-formula_19_divider_formula_19" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i17.jpg"></div></span> denote a heterozygous genotype, where <span data-id="inline-formula_20_divider_formula_20" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i18.jpg"></div></span> and <span data-id="inline-formula_21_divider_formula_21" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i19.jpg"></div></span> each represent one of the length variants present in the data. In the heterozygous case, the values <span data-id="inline-formula_22_divider_formula_22" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i20.jpg"></div></span> and <span data-id="inline-formula_23_divider_formula_23" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i21.jpg"></div></span> are 3D vectors. The values <span data-id="inline-formula_24_divider_formula_24" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i22.jpg"></div></span>and <span data-id="inline-formula_25_divider_formula_25" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i23.jpg"></div></span> represent the number of reads of variants <span data-id="inline-formula_26_divider_formula_26" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i24.jpg"></div></span> and <span data-id="inline-formula_27_divider_formula_27" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i25.jpg"></div></span>, respectively, and <span data-id="inline-formula_28_divider_formula_28" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i26.jpg"></div></span> is the number of reads of any other variant. Values <span data-id="inline-formula_29_divider_formula_29" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i27.jpg"></div></span> and <span data-id="inline-formula_30_divider_formula_30" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i28.jpg"></div></span> represent the probability that a read at this locus is of variant <span data-id="inline-formula_31_divider_formula_31" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i29.jpg"></div></span> and <span data-id="inline-formula_32_divider_formula_32" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i30.jpg"></div></span>, respectively, and <span data-id="inline-formula_33_divider_formula_33" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i31.jpg"></div></span> represents the probability that a read does not represent a true variant (e.g. a sequencing or mapping artifact relative to the genotype <span data-id="inline-formula_34_divider_formula_34" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i32.jpg"></div></span> under consideration). The homozygous genotypes are modelled similarly with 2D vectors: position one refers to the variant supported by the genotype and the second position refers to all other alleles.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_23"><div class="content"><div class="content-node text" data-id="text_23"><div class="content">The vector <span data-id="inline-formula_35_divider_formula_35" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i33.jpg"></div></span> characterizes prior information about the probability vector <span data-id="inline-formula_36_divider_formula_36" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i34.jpg"></div></span>, and <span data-id="inline-formula_37_divider_formula_37" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i35.jpg"></div></span> is formed based on derived error profiles. In the heterozygous case, let <span data-id="inline-formula_38_divider_formula_38" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i36.jpg"></div></span> represent the number of majority allele supporting reads of variant A from the error profiles, and let <span data-id="inline-formula_39_divider_formula_39" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i37.jpg"></div></span> represent the number of non-majority allele supporting reads from the error profiles. Let <span data-id="inline-formula_40_divider_formula_40" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i38.jpg"></div></span> and <span data-id="inline-formula_41_divider_formula_41" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i39.jpg"></div></span> be similarly defined for variant B. Hyperparameters <span data-id="inline-formula_42_divider_formula_42" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i40.jpg"></div></span> and <span data-id="inline-formula_43_divider_formula_43" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i41.jpg"></div></span>. In the homozygous case, hyperparameters <span data-id="inline-formula_44_divider_formula_44" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i42.jpg"></div></span> and <span data-id="inline-formula_45_divider_formula_45" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i43.jpg"></div></span>, where <span data-id="inline-formula_46_divider_formula_46" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i44.jpg"></div></span> and <span data-id="inline-formula_47_divider_formula_47" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i45.jpg"></div></span> are as defined previously. The Bayesian model describing the probability of read error given genotype <span data-id="inline-formula_48_divider_formula_48" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i46.jpg"></div></span> is as follows:</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node formula" data-id="formula_49"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981um1.jpg"></div></div><div class="content-node paragraph" data-id="paragraph_25"><div class="content"><div class="content-node text" data-id="text_25"><div class="content">This model specification ensures that the posterior distribution <span data-id="inline-formula_49_divider_formula_50" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i47.jpg"></div></span> weighs the error profile reads equally to reads from the data. We consider the aforementioned model specification for all genotypes <span data-id="inline-formula_50_divider_formula_51" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i48.jpg"></div></span>. The marginal distribution of the data given each genotype is estimated, and Bayes� rule is used to compute the probability of each genotype given the data. This strategy is a fully Bayesian model selection algorithm, for which each model under consideration corresponds to one of the genotypes suggested by the data. The marginal distribution of the data for a given genotype is</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node formula" data-id="formula_52"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981um2.jpg"></div></div><div class="content-node paragraph" data-id="paragraph_26"><div class="content"><div class="content-node text" data-id="text_26"><div class="content">where <span data-id="inline-formula_51_divider_formula_53" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i49.jpg"></div></span> represents the multinomial beta function. This analytical solution is based on conjugacy results. Bayes� rule is then implemented to invert the above probabilities.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node formula" data-id="formula_54"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981um3.jpg"></div></div><div class="content-node paragraph" data-id="paragraph_27"><div class="content"><div class="content-node text" data-id="text_27"><div class="content">where <span data-id="inline-formula_52_divider_formula_55" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i50.jpg"></div></span> for all <span data-id="inline-formula_53_divider_formula_56" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i51.jpg"></div></span>. The most probable genotype is then selected, provided that the probability of the genotype is &gt;50%.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-2" data-id="heading_12"><div class="content"><span class="text title">RESULTS</span></div><div class="focus-handle"></div></div><div class="content-node heading level-3" data-id="heading_9"><div class="content"><span class="text title">Optimization of read mapping for microsatellite repeat genotyping</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_28"><div class="content"><div class="content-node text" data-id="text_28"><div class="content">The accuracy of repeat genotypes is contingent on the proper mapping of reads to repeat loci. There are many short-read mapping algorithms, and their performance has been widely evaluated (<a href="" data-id="citation_reference_23_divider_article_citation_19" class="annotation citation_reference resource-reference">19</a>). However, mapping algorithms have not yet been evaluated on their ability to accurately map reads composed of low-complexity sequence. We compared the accuracy of several popular tools, including Bowtie2 (<a href="" data-id="citation_reference_24_divider_article_citation_20" class="annotation citation_reference resource-reference">20</a>), BWA (<a href="" data-id="citation_reference_25_divider_article_citation_21" class="annotation citation_reference resource-reference">21</a>), Novoalign (Novocraft Technologies), Stampy (<a href="" data-id="citation_reference_26_divider_article_citation_22" class="annotation citation_reference resource-reference">22</a>) and SMALT (Wellcome Trust Sanger Institute). First, reference repeats were identified from the hg19 reference sequence using a previously described approach (<a href="" data-id="citation_reference_27_divider_article_citation_11" class="annotation citation_reference resource-reference">11</a>). Next, DWGSIM (<a href="http://github.com/nh13/DWGSIM" data-id="link_3_divider_undefined" class="link">http://github.com/nh13/DWGSIM</a>) was used to simulate 100 bp single-end Illumina reads from reference repeats from chromosomes 1 through 7 in human reference sequence, with a simulated coverage of 15�. We specified a 2% uniform sequencing error rate and a 0.1% mutation rate, of which 10% of the mutations were small indels (1�10 bp long) and the remainder were point mutations.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_29"><div class="content"><div class="content-node text" data-id="text_29"><div class="content">Simulated reads were then mapped to the entire human reference sequence. The parameters, runtimes and resource usage for all tools are documented in <a href="http://nar.oxfordjournals.org/cgi/content/full/gks981/DC1" data-id="link_4_divider_undefined" class="link">Supplementary Table S1</a>. The proportion of incorrectly mapped reads is plotted as a function of the proportion of correctly mapped reads in <a href="" data-id="figure_reference_2_divider_figure_2" class="annotation figure_reference resource-reference">Figure 2</a> under different mapping quality thresholds. For each mapping quality threshold, the accuracy rate is the number of correctly mapped reads divided by the total amount of reads simulated. Likewise, the inaccuracy rate is the number of incorrectly mapped reads divided by the total number of mapped reads. Assessment of read alignment is based on the leftmost coordinate of the read. Novoalign maps repeat-containing reads with the lowest ratio of inaccurate calls to accurate calls. Among the open source tools, Bowtie2-high (Bowtie2 using the highest sensitivity option) is the best mapping method by this metric. Other settings of Bowtie2, along with Stampy and BWA-SW, performed fairly similarly among each other, but not as well as Novoalign or Bowtie2-high.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_30"><div class="content"><div class="content-node text" data-id="text_30"><div class="content"><a href="" data-id="figure_reference_3_divider_html_table_1" class="annotation figure_reference resource-reference">Table 1</a> summarizes the performance of the mapping programs and includes a comparison with the lobSTR method, which eschews conventional short-read mappers in favour of its own method for directly assigning reads to a proper location on the human reference sequence. The table indicates performance cumulatively for all mapping qualities, to enable fair comparison with lobSTR, which does not report mapping quality. Novoalign mapped the greatest number of correctly mapped reads (40 547 527; 93.9%), although SMALT mapped the greatest number of reads overall (41 180 368; 95.4%). LobSTR mapped the fewest number of correctly mapped reads (1 117 142; 2.59%) but also the fewest reads overall (1 118 902; 2.59%). Maximizing read mapping might result in increased numbers of improperly mapped reads; therefore, we also compared mapping methods on the basis of their inaccuracy rate. We found that lobSTR features the lowest inaccuracy rate of 0.16%, followed by Novoalign with an inaccuracy rate of 1.14%. The open-source Bowtie2-high method also performs nearly as well with an inaccuracy rate of 1.18%.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_31"><div class="content"><div class="content-node text" data-id="text_31"><div class="content">Comparing performance cumulatively for all mapping qualities likely underestimates the accuracy of conventional mappers because even minimal filtering by mapping quality can exclude inaccurately mapped reads. For example, filtering reads based on a minimum mapping quality of 4 lowers the Novoalign inaccuracy rate to 0.007%, at the cost of reducing the number of correctly mapped reads to 40 297 714 (93.3%). At a minimum mapping quality of 3, Novoalign features an inaccuracy rate of 0.33% with 40 436 553 (93.6%) correctly mapped reads. Bowtie2-high performs similarly with an inaccuracy rate of 0.03% (40 213 461 or 93.1% correctly mapped reads) with a minimum mapping quality of 2. Filtering reads by mapping quality allows for optimizing the trade-off of maximizing correctly mapped reads while minimizing the incorrectly mapped reads.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-3" data-id="heading_10"><div class="content"><span class="text title">Application of RepeatSeq to whole genomes from a parent�offspring trio</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_32"><div class="content"><div class="content-node text" data-id="text_32"><div class="content">A subset of genotyping error can be inferred by evaluating whether called genotypes from a parent�offspring dataset are consistent with Mendelian inheritance (<a href="" data-id="citation_reference_28_divider_article_citation_23" class="annotation citation_reference resource-reference">23</a>). We applied RepeatSeq to whole genome data generated using the Illumina HiSeq and 100 bp reads, from the CEU trio, a family from Utah with European ancestry (CEU genomes: NA12878, NA12891, NA12892). The average coverage of the mapped and post-processed genomes was 42.7� and this coverage dropped to 33.3� at repeat regions. In testing for consistency with Mendelian inheritance, we considered repeats for which calls were made from all genomes in the trio and for which there was at least a single mapped discordant read at the locus in each genome. RepeatSeq requires at least two reads to make a call and for this threshold, 92.1% of evaluated repeat calls are consistent with Mendelian inheritance (<a href="http://nar.oxfordjournals.org/cgi/content/full/gks981/DC1" data-id="link_5_divider_undefined" class="link">Supplementary Table S2</a>). At a minimum coverage of 9�, the percentage of consistent calls increases to 95.3%, and at 17�, the percentage increase to 98.0%.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_33"><div class="content"><div class="content-node text" data-id="text_33"><div class="content">This consistency test also enables the assessment of the impact of choosing a less informative prior for the genotyping model. We replaced our error profile with a constant rate prior in which we define a single observed concordant reads and no discordant reads for every element of the error profile. This strategy eschews the experimentally derived error profile and instead incorporates weak prior information into our genotyping algorithm. Not surprisingly, the percentage of consistent calls drops substantially. At a minimum of 2, 9 and 17� coverage, the constant rate prior produces calls that are consistent 72.9, 73.7 and 74.5% of the time, respectively (<a href="http://nar.oxfordjournals.org/cgi/content/full/gks981/DC1" data-id="link_6_divider_undefined" class="link">Supplementary Table S2</a>).</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-3" data-id="heading_11"><div class="content"><span class="text title">Application of RepeatSeq to the HG01140 whole genome</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_34"><div class="content"><div class="content-node text" data-id="text_34"><div class="content">We compared the performance of our genotyper with lobSTR, a recently reported tool that is also the only other genotyper tuned for microsatellite repeats. To perform the comparison, a 16� sequenced human genome (HG01140) from the 1000 Genomes Project was mapped against the hg19 reference sequencing using Novoalign, post-processed and passed to both genotyping methods. The FASTQ files for this genome were mapped using Novoalign and realigned using the GATK IndelRealigner. RepeatSeq was then executed using default parameters to generate a list of repeat genotypes. To genotype repeats with lobSTR, we used the available binaries (lobSTR version 1.0.6) and usage guidelines. First, lobSTR built an index for the hg19 reference genome. The same list of repeat regions used by RepeatSeq was indexed for use by lobSTR. Next, lobSTR constructed a new BAM file (based on the same BAM file used by RepeatSeq) containing its alignments for genotype calling. We used default parameters except that we specified genotyping repeats with a unit length of 1�5. The lobSTR-generated BAM file was then passed to the lobSTR allelotyper tool to produce a list of repeat genotypes.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_35"><div class="content"><div class="content-node text" data-id="text_35"><div class="content"><a href="" data-id="figure_reference_4_divider_html_table_2" class="annotation figure_reference resource-reference">Table 2</a> summarizes the comparison of RepeatSeq and lobSTR genotypes. The percentages indicate the fraction of genotypes made from a list of 4 030 980 reference repeat regions. Unlike the earlier comparison using simulated data from chromosomes 1 through 7, this comparison includes all identified hg19 reference repeat regions. RepeatSeq assigned genotypes to 3 604 790 (89.4%) repeats, whereas lobSTR assigned genotypes 106 043 (2.63%) repeats. When the comparison is made by repeat unit size, RepeatSeq assigns genotypes to a similar proportion of repeats for all unit sizes, whereas lobSTR assigned the most genotypes to dinucleotide repeats (64 670; 10.3%). For repeat regions that were genotyped by RepeatSeq and lobSTR, we measured the concordance of the methods or cases in which both methods assigned the same genotype. Genotypes were assigned by both methods for 96 950 repeats. Of these, 85 569 (88.3%) repeat genotypes were the same. The concordance between the methods is lowest for 2mers (83.4%) and increases with unit size, which is expected given that the variability of microsatellite repeats is dependent on the size of the repeated unit sequence. From the 11 381 (11.7%) discordant genotypes, we randomly selected repeat regions and used Sanger sequencing to confirm which method assigned the correct genotype. After sequencing 40 regions, we found that 25 (62.5%) regions were correctly genotyped by RepeatSeq, 4 (10.0%) regions were correctly genotyped by lobSTR and 11 (27.5%) regions were incorrectly genotyped by both methods (<a href="http://nar.oxfordjournals.org/cgi/content/full/gks981/DC1" data-id="link_7_divider_undefined" class="link">Supplementary Table S3</a>).</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_36"><div class="content"><div class="content-node text" data-id="text_36"><div class="content">From the 3 604 790 repeat genotypes assigned by RepeatSeq, lobSTR failed to genotype 3 507 840 (97.3%) regions. This is consistent with lobSTRs greatly reduced number of total and correctly mapped reads for the simulated data (<a href="" data-id="figure_reference_5_divider_html_table_1" class="annotation figure_reference resource-reference">Table 1</a>). However, from the 106 043 repeat genotypes assigned by lobSTR, 9093 repeat genotypes (8.57%) were missed by RepeatSeq. In 5090 (56.0%) of the repeats, lobSTR was able to map more reads to the locus than RepeatSeq had access to, and this likely enabled the assignment of a genotype by lobSTR. In nearly half of these cases (2436 repeats), lobSTR added more than one mapped read to the locus. We also found that lobSTR assigns genotypes if at least a single read is mapped to the locus. RepeatSeq, however, requires at least two mapped reads, as a single read is not sufficient to confidently distinguish a heterozygous locus from a homozygous locus. There were 3819 (42.0%) cases in which lobSTR exclusively assigned a genotype using only a single read. For the remaining repeats genotyped by lobSTR and not RepeatSeq, manual inspection revealed that RepeatSeq failed to genotype many of these repeats because it could not assign a high enough confidence (&gt;50%) to the genotype. Finally, there were 417 097 (10.3%) repeats not genotyped by either method primarily because no reads could be mapped to these regions. In all, RepeatSeq assigns genotypes to &#8764;90% of repeats in the HG01140 genome, and in the cases where both methods make a call, the concordance in genotypes is &#8764;90%. Although we did not exhaustively validate the discordant genotypes, we do not see any evidence to suggest that lobSTR generally assigns genotypes more accurately than RepeatSeq.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-2" data-id="heading_13"><div class="content"><span class="text title">DISCUSSION</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_37"><div class="content"><div class="content-node text" data-id="text_37"><div class="content">Repetitive DNA sequences pose unique challenges to next-generation sequencing technologies. However, these sequences are too important to ignore, in terms of their frequent occurrence in the genome and their biological relevance. RepeatSeq uses properties of reference repeat sequences and the reads that map to them, to best identify reads that contribute to true alleles. To enable accurate RepeatSeq calls, we first evaluated several methods for mapping repeat-containing reads to the human reference sequence. We found that when we filtered reads by mapping quality, Novoalign and Bowtie2 correctly map &gt;93% of simulated repeat-containing reads to the human reference (compared with 3% mapped by lobSTR) while maintaining an inaccuracy rate that is lower than lobSTR. Filtering reads by mapping quality allowed for optimizing the trade-off of maximizing correctly mapped reads while minimizing the incorrectly mapped reads.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_38"><div class="content"><div class="content-node text" data-id="text_38"><div class="content">When we analysed the HG01140 genome from the 1000 Genomes Project, we found that RepeatSeq assigned genotypes to &#8764;90% of the repeats, whereas lobSTR genotyped &lt;3%. The disparity between the methods results likely from the difference in how many reads are mapped by lobSTR and Novoalign. LobSTR did make calls that RepeatSeq missed for 9093 (8.57%) repeats. Of these calls that RepeatSeq missed, 42% were made with the support of a single read. RepeatSeq requires the support of two reads to make calls in diploid genomes. From the remaining cases, RepeatSeq sometimes failed to have high enough genotype probabilities (&gt;50%), and in other cases, lobSTR was able to exclusively make a call likely because it was able to map more reads to the locus. There is no doubt that there are cases in which lobSTR will excel at assigning genotypes, particularly those that exhibit extreme allelic variation from the reference (such as in the case of expanded repeats at the HTT locus in Huntington�s patients); however, based on the number of calls, &#8764;90% concordance in genotypes for repeats in which both methods made a call and our Sanger validation data, RepeatSeq offers the best comprehensive approach for exhaustively genotyping repeats in human genomes.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_39"><div class="content"><div class="content-node text" data-id="text_39"><div class="content">RepeatSeq is multithread and scales well on multi-core processors. An analysis of &#8764;4 million reference repeats in the NA12878 genome (40� post-processed coverage, 100 bp reads) takes 11 h using a single core of an AMD Opteron 6174 CPU and a maximum of 1.65 GB of memory. With 48 cores, the runtime is reduced to 20 min. Runtimes can be further reduced by opting to generate only the VCF output. In addition to outputting genotypes using VCF, RepeatSeq optionally produces a detailed report that includes an annotated alignment of all the reads to the reference sequence for each microsatellite locus. We envision this extended report will be valuable for manually confirming calls and for further refining the method. The use of VCF output ensures the compatibility of RepeatSeq with other variant callers and analysis pipelines. For example, the VCF output of RepeatSeq can be used to augment indel calls from GATKs UnifiedGenotyper to provide a comprehensive and more accurate set of variant calls.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-2" data-id="heading_14"><div class="content"><span class="text title">SUPPLEMENTARY DATA</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_40"><div class="content"><div class="content-node text" data-id="text_40"><div class="content"><a href="http://nar.oxfordjournals.org/cgi/content/full/gks981/DC1" data-id="link_8_divider_undefined" class="link">Supplementary Data</a> are available at NAR Online: Supplementary Tables 1�3.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-2" data-id="heading_15"><div class="content"><span class="text title">FUNDING</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_41"><div class="content"><div class="content-node text" data-id="text_41"><div class="content">[</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_42"><div class="content"><div class="content-node text" data-id="text_42"><div class="content">to D.M.]; </div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_43"><div class="content"><div class="content-node text" data-id="text_43"><div class="content">program (to D.M.). Funding for open access charge: </div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_44"><div class="content"><div class="content-node text" data-id="text_44"><div class="content">.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_45"><div class="content"><div class="content-node text" data-id="text_45"><div class="content"><span data-id="emphasis_8_divider_undefined" class="annotation emphasis">Conflict of interest statement</span>. None declared.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="surface-scrollbar content"><div class="visible-area" style="top: 30.6316224026421px; height: 16.8570249071144px;"></div></div></div><div class="medial-strip"><div class="separator-line"></div><div class="context-toggles"><a title="Content" class="context-toggle toc active"><i class="fa fa-align-left"></i><div class="label">Content</div><span> Content</span></a><a title="Figures" class="context-toggle figures"><i class="fa fa-picture-o"></i><div class="label">Figures</div><span> Figures</span></a><a title="References" class="context-toggle citations"><i class="fa fa-link"></i><div class="label">References</div><span> References</span></a><a title="Glossary" class="context-toggle definitions hidden"><i class="fa fa-book"></i><div class="label">Glossary</div><span> Glossary</span></a><a title="Info" class="context-toggle info"><i class="fa fa-info"></i><div class="label">Info</div><span> Info</span></a></div></div><div class="resources"><div class="panel toc resource-view"><div class="toc"><div class="heading-ref level-1" id="toc_heading_1"><span class="">Abstract</span></div><div class="heading-ref level-1 active" id="toc_heading_4"><span class="">Main Text</span></div><div class="heading-ref level-2" id="toc_heading_5"><span class="">INTRODUCTION</span></div><div class="heading-ref level-2" id="toc_heading_8"><span class="">METHODS</span></div><div class="heading-ref level-3" id="toc_heading_6"><span class="">Program outline</span></div><div class="heading-ref level-3" id="toc_heading_7"><span class="">Probabilistic determination of repeat genotypes</span></div><div class="heading-ref level-2" id="toc_heading_12"><span class="">RESULTS</span></div><div class="heading-ref level-3" id="toc_heading_9"><span class="">Optimization of read mapping for microsatellite repeat genotyping</span></div><div class="heading-ref level-3" id="toc_heading_10"><span class="">Application of RepeatSeq to whole genomes from a parent�offspring trio</span></div><div class="heading-ref level-3" id="toc_heading_11"><span class="">Application of RepeatSeq to the HG01140 whole genome</span></div><div class="heading-ref level-2" id="toc_heading_13"><span class="">DISCUSSION</span></div><div class="heading-ref level-2" id="toc_heading_14"><span class="">SUPPLEMENTARY DATA</span></div><div class="heading-ref level-2" id="toc_heading_15"><span class="">FUNDING</span></div></div></div><div class="panel figures resource-view hidden"><div class="surface resource-view figures"><div class="nodes"><div class="content-node figure" data-id="figure_1"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Figure 1.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="image-wrapper"><a href="./repeatseq_files/gks981f1p.jpg" target="_blank" class=""><img src="./repeatseq_files/gks981f1p.jpg" class=""></a></div><div class="content-node caption" data-id="caption_1"><div class="content"><div class="content-node paragraph" data-id="paragraph_1"><div class="content"><div class="content-node text" data-id="text_1"><div class="content">An outline of the RepeatSeq method. Reads are mapped and realigned, and a set of reads spanning reference repeats are retained. Genotypes are assigned with consideration of the <span data-id="emphasis_3_divider_undefined" class="annotation emphasis">a priori</span> error rate <span data-id="inline-formula_1_divider_formula_1" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i1.jpg"></div></span>, which comes from the appropriate error profile and is used in the prior distribution of allele and error probabilities <span data-id="inline-formula_2_divider_formula_2" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i2.jpg"></div></span>. The probability of each genotype suggested by the data is estimated in a Bayesian fashion, and the most probable genotype among these is called.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div></div><div class="focus-handle"></div></div><div class="content-node figure" data-id="figure_2"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Figure 2.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="image-wrapper"><a href="./repeatseq_files/gks981f2p.jpg" target="_blank" class=""><img src="./repeatseq_files/gks981f2p.jpg" class=""></a></div><div class="content-node caption" data-id="caption_2"><div class="content"><div class="content-node paragraph" data-id="paragraph_2"><div class="content"><div class="content-node text" data-id="text_2"><div class="content">The performance of various methods for mapping reads to reference repeats. Mapping accuracy is determined using simulated 100 bp Illumina reads (with a coverage of 15�) and is assessed by measuring the proportion of incorrectly mapped reads as a function of the proportion of correctly mapped reads under different mapping quality thresholds. Variations of Bowtie2 are fully described as follows: bowtie2 (Bowtie2 with default settings), bowtie2-high (Bowtie2 using the highest sensitivity setting), bowtie2-local (Bowtie2 with default sensitivity and soft-clipping) and bowtie2-local-high (Bowtie2 using the highest sensitivity and soft-clipping).</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div></div><div class="focus-handle"></div></div><div class="content-node html-table" data-id="html_table_1"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Table 1.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="table-wrapper"><table frame="hsides" rules="groups"><thead align="left"><tr><th rowspan="1" colspan="1">Method</th><th rowspan="1" colspan="1">Total mapped <italic>n</italic> (%)</th><th rowspan="1" colspan="1">Correctly mapped <italic>n</italic> (%)</th><th rowspan="1" colspan="1">Incorrectly mapped <italic>n</italic> (%)</th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">lobSTR</td><td rowspan="1" colspan="1">1 118 902 (2.59)</td><td rowspan="1" colspan="1">1 117 142 (2.59)</td><td rowspan="1" colspan="1">1760 (0.16)</td></tr><tr><td rowspan="1" colspan="1">Novoalign</td><td rowspan="1" colspan="1">41 014 531 (95.0)</td><td rowspan="1" colspan="1">40 547 527 (93.9)</td><td rowspan="1" colspan="1">467 004 (1.14)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2</td><td rowspan="1" colspan="1">40 678 703 (94.2)</td><td rowspan="1" colspan="1">40 196 603 (93.0)</td><td rowspan="1" colspan="1">482 100 (1.19)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2-high</td><td rowspan="1" colspan="1">40 946 152 (94.8)</td><td rowspan="1" colspan="1">40 464 488 (93.7)</td><td rowspan="1" colspan="1">481 664 (1.18)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2-local</td><td rowspan="1" colspan="1">40 961 622 (94.9)</td><td rowspan="1" colspan="1">40 448 448 (93.6)</td><td rowspan="1" colspan="1">513 174 (1.25)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2-local-high</td><td rowspan="1" colspan="1">40 975 438 (94.9)</td><td rowspan="1" colspan="1">40 472 990 (93.7)</td><td rowspan="1" colspan="1">502 421 (1.23)</td></tr><tr><td rowspan="1" colspan="1">BWA</td><td rowspan="1" colspan="1">39 390 695 (91.2)</td><td rowspan="1" colspan="1">38 941 969 (90.2)</td><td rowspan="1" colspan="1">448 726 (1.14)</td></tr><tr><td rowspan="1" colspan="1">BWASW</td><td rowspan="1" colspan="1">40 611 633 (94.1)</td><td rowspan="1" colspan="1">40 120 872 (92.9)</td><td rowspan="1" colspan="1">490 761 (1.21)</td></tr><tr><td rowspan="1" colspan="1">SMALT</td><td rowspan="1" colspan="1">41 180 368 (95.4)</td><td rowspan="1" colspan="1">40 491 179 (93.7)</td><td rowspan="1" colspan="1">689 189 (1.67)</td></tr><tr><td rowspan="1" colspan="1">Stampy</td><td rowspan="1" colspan="1">41 004 163 (95.0)</td><td rowspan="1" colspan="1">40 478 030 (93.8)</td><td rowspan="1" colspan="1">526 133 (1.28)</td></tr></tbody></table></div><div class="content-node caption" data-id="caption_3"><div class="content"><div class="content-node paragraph" data-id="paragraph_3"><div class="content"><div class="content-node text" data-id="text_3"><div class="content">Performance of mappers for microsatellite repeat regions<span data-id="superscript_1_divider_undefined" class="annotation superscript"><a href="" data-id="cross_reference_1_divider_gks981-TF1" class="annotation cross_reference resource-reference">a</a></span><a href="" data-id="cross_reference_1_divider_gks981-TF1" class="annotation cross_reference resource-reference"></a></div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="footers"></div></div><div class="focus-handle"></div></div><div class="content-node html-table" data-id="html_table_2"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Table 2.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="table-wrapper"><table frame="hsides" rules="groups"><thead align="left"><tr><th rowspan="1" colspan="1">Comparison</th><th rowspan="1" colspan="1">1 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">2 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">3 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">4 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">5 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">Total <italic>n</italic> (%)</th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">RepeatSeq calls</td><td rowspan="1" colspan="1">1 014 806 (88.0)</td><td rowspan="1" colspan="1">556 727 (89.0)</td><td rowspan="1" colspan="1">680 939 (89.7)</td><td rowspan="1" colspan="1">766 010 (90.6)</td><td rowspan="1" colspan="1">586 308 (90.7)</td><td rowspan="1" colspan="1">3 604 790 (89.4)</td></tr><tr><td rowspan="1" colspan="1">lobSTR calls</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">64 670 (10.3)</td><td rowspan="1" colspan="1">15 722 (2.07)</td><td rowspan="1" colspan="1">17 336 (2.05)</td><td rowspan="1" colspan="1">8315 (1.29)</td><td rowspan="1" colspan="1">106 043 (2.63)</td></tr><tr><td rowspan="1" colspan="1">Concordant call</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">47 987 (7.67)</td><td rowspan="1" colspan="1">14 482 (1.91)</td><td rowspan="1" colspan="1">15 430 (1.82)</td><td rowspan="1" colspan="1">7670 (1.19)</td><td rowspan="1" colspan="1">85 569 (2.12)</td></tr><tr><td rowspan="1" colspan="1">Discordant call</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">9538 (1.52)</td><td rowspan="1" colspan="1">624 (0.08)</td><td rowspan="1" colspan="1">946 (0.11)</td><td rowspan="1" colspan="1">273 (0.04)</td><td rowspan="1" colspan="1">11 381 (0.28)</td></tr><tr><td rowspan="1" colspan="1">RepeatSeq call, lobSTR N</td><td rowspan="1" colspan="1">1 014 806 (88.0)</td><td rowspan="1" colspan="1">499 202 (79.8)</td><td rowspan="1" colspan="1">665 833 (87.7)</td><td rowspan="1" colspan="1">749 634 (88.6)</td><td rowspan="1" colspan="1">578 365 (89.5)</td><td rowspan="1" colspan="1">3 507 840 (87.0)</td></tr><tr><td rowspan="1" colspan="1">lobSTR call, RepeatSeq N</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">7145 (1.14)</td><td rowspan="1" colspan="1">616 (0.08)</td><td rowspan="1" colspan="1">960 (0.11)</td><td rowspan="1" colspan="1">372 (0.06)</td><td rowspan="1" colspan="1">9093 (0.23)</td></tr><tr><td rowspan="1" colspan="1">RepeatSeq N, lobSTR N</td><td rowspan="1" colspan="1">138 769 (12.0)</td><td rowspan="1" colspan="1">61 800 (9.88)</td><td rowspan="1" colspan="1">77 758 (10.2)</td><td rowspan="1" colspan="1">78 922 (9.33)</td><td rowspan="1" colspan="1">59 848 (9.26)</td><td rowspan="1" colspan="1">417 097 (10.3)</td></tr></tbody></table></div><div class="content-node caption" data-id="caption_4"><div class="content"><div class="content-node paragraph" data-id="paragraph_4"><div class="content"><div class="content-node text" data-id="text_4"><div class="content">Comparison of RepeatSeq and lobSTR microsatellite calls<span data-id="superscript_2_divider_undefined" class="annotation superscript"><a href="" data-id="cross_reference_2_divider_gks981-TF2" class="annotation cross_reference resource-reference">a</a></span><a href="" data-id="cross_reference_2_divider_gks981-TF2" class="annotation cross_reference resource-reference"></a></div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="footers"></div></div><div class="focus-handle"></div></div><div class="content-node supplement" data-id="supplement_1"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource"></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="content-node caption" data-id="caption_5"><div class="content"><div class="content-node paragraph caption-title" data-id="paragraph_5"><div class="content"><div class="content-node text" data-id="text_5"><div class="content">Supplementary Data</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="file"><a href="file:///Users/gareth/Google%20Drive/lens-n/supp_41_1_e32__index.html" class=""><i class="fa fa-download"> Download</i></a></div></div><div class="focus-handle"></div></div><div class="content-node figure" data-id="figure_3"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Figure 1.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="image-wrapper"><a href="file:///Users/gareth/Google%20Drive/lens-n/gks981f1p" target="_blank" class=""><img src="file:///Users/gareth/Google%20Drive/lens-n/gks981f1p" class=""></a></div><div class="content-node caption" data-id="caption_6"><div class="content"><div class="content-node paragraph" data-id="paragraph_6"><div class="content"><div class="content-node text" data-id="text_6"><div class="content">An outline of the RepeatSeq method. Reads are mapped and realigned, and a set of reads spanning reference repeats are retained. Genotypes are assigned with consideration of the <span data-id="emphasis_4_divider_undefined" class="annotation emphasis">a priori</span> error rate <span data-id="inline-formula_3_divider_formula_3" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i1.jpg"></div></span>, which comes from the appropriate error profile and is used in the prior distribution of allele and error probabilities <span data-id="inline-formula_4_divider_formula_4" class="annotation inline-formula"><div class="MathJax_Preview"><img src="./repeatseq_files/gks981i2.jpg"></div></span>. The probability of each genotype suggested by the data is estimated in a Bayesian fashion, and the most probable genotype among these is called.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div></div><div class="focus-handle"></div></div><div class="content-node figure" data-id="figure_4"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Figure 2.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="image-wrapper"><a href="file:///Users/gareth/Google%20Drive/lens-n/gks981f2p" target="_blank" class=""><img src="file:///Users/gareth/Google%20Drive/lens-n/gks981f2p" class=""></a></div><div class="content-node caption" data-id="caption_7"><div class="content"><div class="content-node paragraph" data-id="paragraph_7"><div class="content"><div class="content-node text" data-id="text_7"><div class="content">The performance of various methods for mapping reads to reference repeats. Mapping accuracy is determined using simulated 100 bp Illumina reads (with a coverage of 15�) and is assessed by measuring the proportion of incorrectly mapped reads as a function of the proportion of correctly mapped reads under different mapping quality thresholds. Variations of Bowtie2 are fully described as follows: bowtie2 (Bowtie2 with default settings), bowtie2-high (Bowtie2 using the highest sensitivity setting), bowtie2-local (Bowtie2 with default sensitivity and soft-clipping) and bowtie2-local-high (Bowtie2 using the highest sensitivity and soft-clipping).</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div></div><div class="focus-handle"></div></div><div class="content-node html-table" data-id="html_table_3"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Table 1.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="table-wrapper"><table frame="hsides" rules="groups"><thead align="left"><tr><th rowspan="1" colspan="1">Method</th><th rowspan="1" colspan="1">Total mapped <italic>n</italic> (%)</th><th rowspan="1" colspan="1">Correctly mapped <italic>n</italic> (%)</th><th rowspan="1" colspan="1">Incorrectly mapped <italic>n</italic> (%)</th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">lobSTR</td><td rowspan="1" colspan="1">1 118 902 (2.59)</td><td rowspan="1" colspan="1">1 117 142 (2.59)</td><td rowspan="1" colspan="1">1760 (0.16)</td></tr><tr><td rowspan="1" colspan="1">Novoalign</td><td rowspan="1" colspan="1">41 014 531 (95.0)</td><td rowspan="1" colspan="1">40 547 527 (93.9)</td><td rowspan="1" colspan="1">467 004 (1.14)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2</td><td rowspan="1" colspan="1">40 678 703 (94.2)</td><td rowspan="1" colspan="1">40 196 603 (93.0)</td><td rowspan="1" colspan="1">482 100 (1.19)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2-high</td><td rowspan="1" colspan="1">40 946 152 (94.8)</td><td rowspan="1" colspan="1">40 464 488 (93.7)</td><td rowspan="1" colspan="1">481 664 (1.18)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2-local</td><td rowspan="1" colspan="1">40 961 622 (94.9)</td><td rowspan="1" colspan="1">40 448 448 (93.6)</td><td rowspan="1" colspan="1">513 174 (1.25)</td></tr><tr><td rowspan="1" colspan="1">Bowtie2-local-high</td><td rowspan="1" colspan="1">40 975 438 (94.9)</td><td rowspan="1" colspan="1">40 472 990 (93.7)</td><td rowspan="1" colspan="1">502 421 (1.23)</td></tr><tr><td rowspan="1" colspan="1">BWA</td><td rowspan="1" colspan="1">39 390 695 (91.2)</td><td rowspan="1" colspan="1">38 941 969 (90.2)</td><td rowspan="1" colspan="1">448 726 (1.14)</td></tr><tr><td rowspan="1" colspan="1">BWASW</td><td rowspan="1" colspan="1">40 611 633 (94.1)</td><td rowspan="1" colspan="1">40 120 872 (92.9)</td><td rowspan="1" colspan="1">490 761 (1.21)</td></tr><tr><td rowspan="1" colspan="1">SMALT</td><td rowspan="1" colspan="1">41 180 368 (95.4)</td><td rowspan="1" colspan="1">40 491 179 (93.7)</td><td rowspan="1" colspan="1">689 189 (1.67)</td></tr><tr><td rowspan="1" colspan="1">Stampy</td><td rowspan="1" colspan="1">41 004 163 (95.0)</td><td rowspan="1" colspan="1">40 478 030 (93.8)</td><td rowspan="1" colspan="1">526 133 (1.28)</td></tr></tbody></table></div><div class="content-node caption" data-id="caption_8"><div class="content"><div class="content-node paragraph" data-id="paragraph_8"><div class="content"><div class="content-node text" data-id="text_8"><div class="content">Performance of mappers for microsatellite repeat regions<span data-id="superscript_3_divider_undefined" class="annotation superscript"><a href="" data-id="cross_reference_3_divider_gks981-TF1" class="annotation cross_reference resource-reference">a</a></span><a href="" data-id="cross_reference_3_divider_gks981-TF1" class="annotation cross_reference resource-reference"></a></div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="footers"></div></div><div class="focus-handle"></div></div><div class="content-node html-table" data-id="html_table_4"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Table 2.</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-fullscreen"><i class="fa fa-expand"></i><i class="fa fa-compress"></i></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="table-wrapper"><table frame="hsides" rules="groups"><thead align="left"><tr><th rowspan="1" colspan="1">Comparison</th><th rowspan="1" colspan="1">1 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">2 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">3 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">4 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">5 <italic>n</italic> (%)</th><th rowspan="1" colspan="1">Total <italic>n</italic> (%)</th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">RepeatSeq calls</td><td rowspan="1" colspan="1">1 014 806 (88.0)</td><td rowspan="1" colspan="1">556 727 (89.0)</td><td rowspan="1" colspan="1">680 939 (89.7)</td><td rowspan="1" colspan="1">766 010 (90.6)</td><td rowspan="1" colspan="1">586 308 (90.7)</td><td rowspan="1" colspan="1">3 604 790 (89.4)</td></tr><tr><td rowspan="1" colspan="1">lobSTR calls</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">64 670 (10.3)</td><td rowspan="1" colspan="1">15 722 (2.07)</td><td rowspan="1" colspan="1">17 336 (2.05)</td><td rowspan="1" colspan="1">8315 (1.29)</td><td rowspan="1" colspan="1">106 043 (2.63)</td></tr><tr><td rowspan="1" colspan="1">Concordant call</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">47 987 (7.67)</td><td rowspan="1" colspan="1">14 482 (1.91)</td><td rowspan="1" colspan="1">15 430 (1.82)</td><td rowspan="1" colspan="1">7670 (1.19)</td><td rowspan="1" colspan="1">85 569 (2.12)</td></tr><tr><td rowspan="1" colspan="1">Discordant call</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">9538 (1.52)</td><td rowspan="1" colspan="1">624 (0.08)</td><td rowspan="1" colspan="1">946 (0.11)</td><td rowspan="1" colspan="1">273 (0.04)</td><td rowspan="1" colspan="1">11 381 (0.28)</td></tr><tr><td rowspan="1" colspan="1">RepeatSeq call, lobSTR N</td><td rowspan="1" colspan="1">1 014 806 (88.0)</td><td rowspan="1" colspan="1">499 202 (79.8)</td><td rowspan="1" colspan="1">665 833 (87.7)</td><td rowspan="1" colspan="1">749 634 (88.6)</td><td rowspan="1" colspan="1">578 365 (89.5)</td><td rowspan="1" colspan="1">3 507 840 (87.0)</td></tr><tr><td rowspan="1" colspan="1">lobSTR call, RepeatSeq N</td><td rowspan="1" colspan="1">N</td><td rowspan="1" colspan="1">7145 (1.14)</td><td rowspan="1" colspan="1">616 (0.08)</td><td rowspan="1" colspan="1">960 (0.11)</td><td rowspan="1" colspan="1">372 (0.06)</td><td rowspan="1" colspan="1">9093 (0.23)</td></tr><tr><td rowspan="1" colspan="1">RepeatSeq N, lobSTR N</td><td rowspan="1" colspan="1">138 769 (12.0)</td><td rowspan="1" colspan="1">61 800 (9.88)</td><td rowspan="1" colspan="1">77 758 (10.2)</td><td rowspan="1" colspan="1">78 922 (9.33)</td><td rowspan="1" colspan="1">59 848 (9.26)</td><td rowspan="1" colspan="1">417 097 (10.3)</td></tr></tbody></table></div><div class="content-node caption" data-id="caption_9"><div class="content"><div class="content-node paragraph" data-id="paragraph_9"><div class="content"><div class="content-node text" data-id="text_9"><div class="content">Comparison of RepeatSeq and lobSTR microsatellite calls<span data-id="superscript_4_divider_undefined" class="annotation superscript"><a href="" data-id="cross_reference_4_divider_gks981-TF2" class="annotation cross_reference resource-reference">a</a></span><a href="" data-id="cross_reference_4_divider_gks981-TF2" class="annotation cross_reference resource-reference"></a></div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="footers"></div></div><div class="focus-handle"></div></div><div class="content-node supplement" data-id="supplement_2"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource"></a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="content-node caption" data-id="caption_10"><div class="content"><div class="content-node paragraph caption-title" data-id="paragraph_10"><div class="content"><div class="content-node text" data-id="text_10"><div class="content">Supplementary Data</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div></div><div class="file"><a href="file:///Users/gareth/Google%20Drive/lens-n/supp_41_1_e32__index.html" class=""><i class="fa fa-download"> Download</i></a></div></div><div class="focus-handle"></div></div></div></div><div class="surface-scrollbar figures hidden"><div class="visible-area"></div></div></div><div class="panel citations resource-view hidden"><div class="surface resource-view citations"><div class="nodes"><div class="content-node citation" data-id="article_citation_1"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">1 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Expandable DNA repeats and human disease</span><div class="authors">SM Mirkin</div><div class="source">Nature, 447: 932-940, 2007</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_2"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">2 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Repeat instability as the basis for human diseases and as a potential target for therapy</span><div class="authors">A Lopez Castel, JD Cleary, CE Pearson</div><div class="source">Nat. Rev. Mol. Cell Biol., 11: 165-170, 2010</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_3"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">3 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Unstable nucleotide repeat minireview series: a molecular biography of unstable repeat disorders</span><div class="authors">HT Orr</div><div class="source">J. Biol. Chem., 284: 2009</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_4"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">4 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">The other trinucleotide repeat: polyalanine expansion disorders</span><div class="authors">A Albrecht, S Mundlos</div><div class="source">Curr. Opin. Genet. Dev., 15: 285-293, 2005</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_5"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">5 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Variable tandem repeats accelerate evolution of coding and regulatory sequences</span><div class="authors">R Gemayel, MD Vinces, M Legendre, KJ Verstrepen</div><div class="source">Annu. Rev. Genet., 44: 445-477, 2010</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_6"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">6 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Stress-induced modulators of repeat instability and genome evolution</span><div class="authors">NC Fonville, RM Ward, D Mittelman</div><div class="source">J. Mol. Microbiol. Biotechnol., 21: 36-44, 2011</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_7"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">7 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Exposure to anticancer drugs can result in transgenerational genomic instability in mice</span><div class="authors">CD Glen, YE Dubrova</div><div class="source">Proc. Natl. Acad. Sci. USA, 109: 2984-2988, 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_8"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">8 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Multiple mutations and cancer</span><div class="authors">LA Loeb, KR Loeb, JP Anderson</div><div class="source">Proc. Natl. Acad. Sci. USA, 100: 776-781, 2003</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_9"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">9 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Developing criteria and data to determine best options for expanding the core CODIS loci</span><div class="authors">J Ge, A Eisenberg, B Budowle</div><div class="source">Investig. Genet., 3: 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_10"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">10 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">lobSTR: a short tandem repeat profiler for personal genomes</span><div class="authors">M Gymrek, D Golan, S Rosset, Y Erlich</div><div class="source">Genome Res., 22: 1154-1162, 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_11"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">11 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Analysis of microsatellite variation in <span data-id="emphasis_1_divider_undefined" class="annotation emphasis">Drosophila melanogaster</span> with population-scale genome sequencing</span><div class="authors">JW Fondon, III, A Martin, S Richards, RA Gibbs, D Mittelman</div><div class="source">PLoS One, 7: 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_12"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">12 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">The <span data-id="emphasis_2_divider_undefined" class="annotation emphasis">Drosophila melanogaster</span> Genetic Reference Panel</span><div class="authors">TFC Mackay, S Richards, EA Stone, A Barbadilla, JF Ayroles, D Zhu, S Casillas, Y Han, MM Magwire, JM Cridland</div><div class="source">Nature, 482: 173-178, 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_13"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">13 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</span><div class="authors">MA DePristo, E Banks, R Poplin, KV Garimella, JR Maguire, C Hartl, AA Philippakis, G del Angel, MA Rivas, M Hanna</div><div class="source">Nat. Genet., 43: 491-498, 2011</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_14"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">14 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">An integrative variant analysis suite for whole exome next-generation sequencing data</span><div class="authors">D Challis, J Yu, US Evani, AR Jackson, S Paithankar, C Coarfa, A Milosavljevic, RA Gibbs, F Yu</div><div class="source">BMC Bioinformatics, 13: 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_15"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">15 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Sequence-based estimation of minisatellite and microsatellite repeat variability</span><div class="authors">M Legendre, N Pochet, T Pak, KJ Verstrepen</div><div class="source">Genome Res., 17: 1787-1796, 2007</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_17"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">17 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">The Sequence Alignment/Map format and SAMtools</span><div class="authors">H Li, B Handsaker, A Wysoker, T Fennell, J Ruan, N Homer, G Marth, G Abecasis, R Durbin</div><div class="source">Bioinformatics, 25: 2078-2079, 2009</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_18"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">18 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">The variant call format and VCFtools</span><div class="authors">P Danecek, A Auton, G Abecasis, CA Albers, E Banks, MA DePristo, RE Handsaker, G Lunter, GT Marth, ST Sherry</div><div class="source">Bioinformatics, 27: 2156-2158, 2011</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_19"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">19 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">A survey of sequence alignment algorithms for next-generation sequencing</span><div class="authors">H Li, N Homer</div><div class="source">Brief. Bioinform., 11: 473-483, 2010</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_20"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">20 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Fast gapped-read alignment with Bowtie 2</span><div class="authors">B Langmead, SL Salzberg</div><div class="source">Nat. Methods., 9: 357-359, 2012</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_21"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">21 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Fast and accurate long-read alignment with Burrows-Wheeler transform</span><div class="authors">H Li, R Durbin</div><div class="source">Bioinformatics, 26: 589-595, 2010</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_22"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">22 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads</span><div class="authors">G Lunter, M Goodson</div><div class="source">Genome Res., 21: 936-939, 2011</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div><div class="content-node citation" data-id="article_citation_23"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">23 - Citation</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><span class="text title">Probability of detection of genotyping errors and mutations as inheritance inconsistencies in nuclear-family data</span><div class="authors">JA Douglas, AD Skol, M Boehnke</div><div class="source">Am. J. Hum. Genet., 70: 487-495, 2002</div><div class="citation-urls"></div></div><div class="focus-handle"></div></div></div></div><div class="surface-scrollbar citations hidden"><div class="visible-area"></div></div></div><div class="panel definitions resource-view hidden"><div class="surface resource-view definitions"><div class="nodes"></div></div><div class="surface-scrollbar definitions hidden"></div></div><div class="panel info resource-view hidden"><div class="surface resource-view info"><div class="nodes"><div class="content-node publication-info" data-id="publication_info"><div class="content"><div class="meta-data"><div class="doi container"><div class="label">DOI</div><div class="value"><a href="http://dx.doi.org/10.1093/nar/gks981" target="_blank" class="">10.1093/nar/gks981</a></div></div><div class="dates">The article was received on <b class="">Mon, 09 Jul 2012</b>, revised on <b class="">Tue, 25 Sep 2012</b>, accepted on <b class="">Thu, 27 Sep 2012</b>, and published on <b class="">January 2013</b>.</div></div><div class="content-node paragraph" data-id="articleinfo"><div class="content"><div class="content-node heading level-3" data-id="heading_2"><div class="content"><span class="text title">Acknowledgements</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_12"><div class="content"><div class="content-node text" data-id="text_12"><div class="content">The authors would like to thank Leanna House, Sunil Kamalakar, R. Matthew Ward and members of the Mittelman Lab for helpful comments and suggestions.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node heading level-3" data-id="heading_3"><div class="content"><span class="text title">Copyright &amp; License</span></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_13"><div class="content"><div class="content-node text" data-id="text_13"><div class="content">� The Author(s) 2012. Published by Oxford University Press.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node paragraph" data-id="paragraph_14"><div class="content"><div class="content-node text" data-id="text_14"><div class="content">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by-nc/3.0/" data-id="link_2_divider_undefined" class="link">http://creativecommons.org/licenses/by-nc/3.0/</a>), which permits non-commercial reuse, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div></div><div class="focus-handle"></div></div><div class="content-node contributor" data-id="contributor_1"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Author</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="contributor-name">Gareth Highnam</div><div class="affiliations"><div class="affiliation"></div></div></div><div class="focus-handle"></div></div><div class="content-node contributor" data-id="contributor_2"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Author</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="contributor-name">Christopher Franck</div><div class="affiliations"><div class="affiliation"></div><div class="affiliation"></div></div></div><div class="focus-handle"></div></div><div class="content-node contributor" data-id="contributor_3"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Author</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="contributor-name">Andy Martin</div><div class="affiliations"><div class="affiliation"></div></div></div><div class="focus-handle"></div></div><div class="content-node contributor" data-id="contributor_4"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Author</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="contributor-name">Calvin Stephens</div><div class="affiliations"><div class="affiliation"></div></div></div><div class="focus-handle"></div></div><div class="content-node contributor" data-id="contributor_5"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Author</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="contributor-name">Ashwin Puthige</div><div class="affiliations"><div class="affiliation"></div></div></div><div class="focus-handle"></div></div><div class="content-node contributor" data-id="contributor_6"><div class="resource-header"><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="name action-toggle-resource">Author</a><a href="file:///Users/gareth/Google%20Drive/lens-n/index.html#" class="toggle-res action-toggle-resource"><i class="fa fa-eye"></i><i class="fa fa-eye-slash"></i></a></div><div class="content"><div class="contributor-name">David Mittelman</div><div class="affiliations"><div class="affiliation"></div><div class="affiliation"></div></div><div class="label">For correspondence</div><div class="emails"><a href="mailto:david.mittelman@vt.edu" class="">david.mittelman@vt.edu</a></div></div><div class="focus-handle"></div></div></div></div><div class="surface-scrollbar info hidden"><div class="visible-area"></div></div></div></div></div></div></div></body></html>
